pkg/controller/rmdworkload/rmdworkload_controller_test.go

```diff
@@ -473,6 +473,437 @@ func TestRmdWorkloadControllerReconcile(t *testing.T) {
 	}
 }
 
+func TestFindObseleteWorkloads(t *testing.T) {
```

Comment:
A general thought on this - would we be able to test with multiple RmdNodeStates and multiple RmdPods? It would also mean creating listeners at separate addresses (for each `rmdPod`) with each having a separate `[]rmdtypes.RDTWorkload` returned.

Reply:
I can definitely give it a go! 😁